fixtures/tests/_fixtures/test_popen.py

```diff
@@ -114,6 +114,15 @@ def test_kill(self):
         proc = FakeProcess({}, {})
         self.assertIs(None, proc.kill())
 
+    def test_poll(self):
+        proc = FakeProcess({}, {})
+        self.assertIs(None, proc.poll())
```

Comment:
I think this needs documenting in the docs for this  docstring and README.md/rst/txt I forget :)